languages/python/django-oso/django_oso/partial.py

```diff
@@ -22,14 +24,193 @@
     "Lt": lambda f, v: Q(**{f"{f}__lt": v}),
 }
 
-VARIABLES = {}
+# So that 0 < field can be written
+# as field > 0 instead
+REFLECTED_COMPARISONS = {
+    "Unify": COMPARISONS["Unify"],
+    "Eq": COMPARISONS["Eq"],
+    "Neq": COMPARISONS["Neq"],
+    "Lt": COMPARISONS["Gt"],
+    "Leq": COMPARISONS["Geq"],
+    "Gt": COMPARISONS["Lt"],
+    "Geq": COMPARISONS["Leq"],
+}
 
 
+# TODO: put in comparisons dict above?
```

Comment:
We didn't want to include `In` in the comparisons dict because `in` operations aren't really comparisons. We thought about renaming `COMPARISONS` to a few broader alternatives (that I can't remember at the moment), but ultimately decided against it.

Reply:
Yeah that's what I thought